background/score/activities.go

```diff
@@ -175,12 +187,15 @@ func (s *ScoreUpdateWorker) RefreshLocationStateActivity(ctx context.Context, ac
 		lastSpikeDay := time.Date(lastSpikeUpdate.Year(), lastSpikeUpdate.Month(), lastSpikeUpdate.Day(), 0, 0, 0, 0, accountLocation)
 
 		if currentSpikeLength := len(metric.Details.Symptoms.LastSpikeList); currentSpikeLength > 0 {
-			if accountToday.Sub(lastSpikeDay) == 0 { // spike in the same day
+			if spikeDayDelta := accountToday.Sub(lastSpikeDay); spikeDayDelta == 0 { // spike in the same day
 				if currentSpikeLength > len(profile.Metric.Details.Symptoms.LastSpikeList) {
 					symptomsSpikeAccounts = append(symptomsSpikeAccounts, profile.AccountNumber)
 				}
-			} else {
+			} else if spikeDayDelta > 1 {
 				symptomsSpikeAccounts = append(symptomsSpikeAccounts, profile.AccountNumber)
+			} else {
```

Comment:
Only when `spikeDayDelta == 1` does it fall through to the last else clause. So if I get notified yesterday, I will receive nothing today?

Reply:
This should be `spikeDayDelta > 0`.